code/game/objects/items/stunbaton.dm

```diff
@@ -158,6 +158,10 @@
 	toggle_on(user)
 
 /obj/item/melee/baton/proc/toggle_on(mob/user)
+	if(!turned_on && istype(user.mind?.martial_art, /datum/martial_art/nemesis)) //I really, REALLY don't want Nemesis Kick users to be able to use stunbatons
```

Comment:
okay, they'll just use a classic baton conversion kit and use classic batons instead

Reply:
Detective batons don't stun, they just knock you down and deal stam damage